packages/typespec-client-generator-core/test/types.test.ts

```diff
@@ -2656,14 +2654,14 @@ describe("typespec-client-generator-core: types", () => {
       const models = runner.context.experimental_sdkPackage.models;
       strictEqual(models.length, 1);
       const scopes = (models[0] as SdkModelType).properties.find(
-        (x) => x.nameInClient === "scopes"
```

Comment:
if we are going to deprecate in a transition we should be asserting both are the same in all these cases until / unless we actually remove it completely.

Otherwise we aren't validating that the old property still works.